content/MoM/base/tiles.ini

```diff
@@ -10,6 +10,9 @@ image="../ffg/img/Tile_Alley_Corner_1_MAD20.dds"
 [TileSideAlleyCorner2]
 image="../ffg/img/Tile_Alley_Corner_2_MAD20.dds"
 
+[TileSideAlleyEnd]
+image="img/Tile_Alley_End_MAD20.jpg"
```

Comment:
Didn't understand this comment, I added all missing tiles to the ini files. 
The "rare" one should be the Library because it's still on the ffg folder. It was the only one I didn't have to create tile, only edited tiles.ini.